### PR TITLE
Create constant for ignored devices

### PR DESCRIPTION
**Pull request by tetienne**

### CODE REVIEW

File: custom_components/tahoma/const.py [hunk 1]
```diff
@@ -9,6 +9,12 @@
 
 DOMAIN = "tahoma"
 
+TAHOMA_NAMES_IGNORED = [
+    "ogp:Bridge",
+    "internal:PodV2Component",
+    "internal:TSKAlarmComponent",
```
Comment (tetienne) [inline custom_components/tahoma/const.py:15]: To check if we can not use the widget or ui class.

Comment (iMicknl) [reply custom_components/tahoma/const.py:15]: That was exactly my thought as well! Only for alarm we would need to scope a bit better, but perhaps there is a widget for internal alarms.

Comment (iMicknl) [reply custom_components/tahoma/const.py:15]:
```
"qualifiedName": "internal:TSKAlarmComponent",
"widgetName": "TSKAlarmController",
"uiClass": "Alarm",
```

```
"qualifiedName": "internal:PodV2Component",
"widgetName": "Pod",
"uiClass": "Pod",
```

```
"qualifiedName": "ogp:Bridge",
"widgetName": "DynamicBridge",
"uiClass": "ProtocolGateway",
```

Best would be to match on `uiClass` and/or `widgetName`. For the Alarm, we would need to match on widgetName is TSKAlarmController.

Comment (iMicknl) [reply custom_components/tahoma/const.py]: @tetienne I took the liberty to make a small change to your branch myself. :-)

**Pull request merged**